src/LayerPlan.cpp

```diff
@@ -146,19 +146,28 @@ Polygons LayerPlan::computeCombBoundaryInside(CombingMode combing_mode)
         for (const SliceMeshStorage& mesh : storage.meshes)
         {
             const SliceLayer& layer = mesh.layers[layer_nr];
+            const bool is_infill_mesh = mesh.getSettingBoolean("infill_mesh");
```

Comment:
I think your solution isn't the right one.
I think the check which *was* on line 158 should be performed here.
So right below this line it should say:
```
if (is_infill_mesh)
{
    continue;
}
```